google_guest_agent/events/metadata/metadata.go

```diff
@@ -0,0 +1,80 @@
+//  Copyright 2023 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package metadata implement the metadata events watcher.
+package metadata
+
+import (
+	"context"
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/GoogleCloudPlatform/guest-agent/metadata"
+	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
+)
+
+const (
+	// WatcherID is the metadata watcher's ID.
+	WatcherID = "metadata-watcher"
+)
+
+var (
+	// arbitrarily defined wait duration(keeps behavioral backward compatibility).
+	retryWaitDuration = 5 * time.Second
```

Comment:
it doesn't seem to be used anywhere, was it intended to use on line 59?

Reply:
Correct, line 59 will get updated. Thank you.